### PR TITLE
[NUI] Fix build warning (CA1714)

### PR DESCRIPTION
**Pull request by sparrow74**
### Description of Change ###

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1714

### CODE REVIEW

**Comment by huiyueun** [comment]
If you add the changed enum name, it would be better to change it for the existing use case as well.

**Comment by sparrow74** [comment]
@huiyueun 
Thank you for your review. I sent you the mail about this patch.
If this patch is added, we should change more public APIs.

File: src/Tizen.NUI/src/public/Layouting/LayoutTransition.cs [hunk 1]
```diff
@@ -69,6 +69,57 @@ public enum TransitionCondition
 
         ChangeOnRemove = 16
     }
+    
+    /// <summary>
+    /// The conditions for transitions.
+    /// </summary>
+    /// <since_tizen> 6.5 </since_tizen>
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Layouting/LayoutTransition.cs:76]: could you remove <since_tizen> tags?
this tag is supposed to be added when "Add ACR" is accepted.
this looks hidden-API, so I don't think versioning tag is needed.
thank you.

File: src/Tizen.NUI/src/public/Layouting/LayoutTransition.cs [hunk 2]
```diff
@@ -69,6 +69,57 @@ public enum TransitionCondition
...
+        /// Animation when an item changes due to a  being added.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        ChangeOnAdd = 8,
+
+        /// <summary>
+        /// Animation when an item changes due to a  being removed.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Layouting/LayoutTransition.cs:119]: same as above

**Comment by sparrow74** [comment]
@dongsug-song 
I think we have one decisions about this patch is alive or not. 
Because if the new enum is added, the public APIs should be changed and need the ACRs.
In addition, this enum and the related public APIs were added in Tizen 6.0. 
So, we will see this build warning

**Pull request closed without merging**